packages/manager/modules/vps/src/dashboard/migrate/migrate.controller.js

```diff
@@ -44,16 +44,21 @@ export default class {
 
   getRenewablePrice() {
     return this.ovhManagerProductOffersService.constructor.getUniquePricingOfCapacity(
-      this.newPlan?.pricings,
+      this.newPlans?.pricings,
       pricingConstants.PRICING_CAPACITIES.RENEW,
     ).price;
   }
 
-  getStorageCapacity() {
-    return this.newPlan.configuration?.storage?.disks[0]?.capacity || 0;
-  }
-
   formatMemory(memory) {
     return memory / this.MEMORY_MULTIPLE;
   }
+
+  getStorageCapacity = (newPlan) => {
+    return newPlan.configuration?.storage?.disks[0]?.capacity || 0;
+  };
+
+  getOriginalPriceWithoutTax = (planCode, newPrices) => {
+    return newPrices.filter((price) => price.newPlan === planCode)[0].price
```

Comment:
Why always take the first element ?

Reply:
HI Marie, newPrice will always return only one value because it will have only one entry with respect to the planCode